src/frontend/js/components/SignContractButton/index.omniscientOrders.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import { PropsWithChildren } from 'react';
```

Comment:
In commit [25f4f74c9ac48c4af9732e25f10546b8388e1247](https://github.com/openfun/richie/commit/25f4f74c9ac48c4af9732e25f10546b8388e1247) message

```diff
-Also, as it'll always need it's ContractFrame, it now includes it.
+Also, as it'll always need its ContractFrame, it now includes it.

```